benchmark/config/snakemake/config.yaml

```
@@ -0,0 +1,9 @@
+"seed": 12345
+"population_id": 0
+"num_samples_per_population": [100, 100, 100]
+"replicates": 100
+"species": "HomSap"
+"model": "OutOfAfrica_3G09"
+"genetic_map": "HapmapII_GRCh37"
+"chrm_list": "chr21"
+"output_dir": "/xdisk/rgutenk/xinhuang/projects/stdpopsim2/analysis2/benchmark/results"
```

TODO: change to relative directory (probably, relative to the root of this repository?)
```suggestion
"output_dir": "benchmark/results"
```